### PR TITLE
Upgrade to Retype docs

### PR DESCRIPTION
**Pull request by geoffreymcgill**
As an alternative to GitBook, I thought I would try upgrading to Retype. Seems to have gone well.

You can see a live demo at:

https://geoffreymcgill.github.io/paaster-client/

I'll delete that site after 24 (or sooner). Just wanted to confirm that publishing to GitHub Pages was working correctly.

If this PR is accepted/merged, you can enable your own GitHub Pages hosting using the following guide:

https://retype.com/hosting/github-pages/#step-2-configure-github-pages

Basically, you just need to make some simple change to enable GitHub Pages hosting. Choose the `retype` branch on the following page and in a few moments your site will be published to GitHub Pages. See:

https://github.com/WardPearce/paaster-client/settings/pages

A few benefits with using Retype:

1. Self hosted
2. Free
3. Dark mode
4. Mobile view is sweet
5. Lightning fast builds
6. [Components](https://retype.com/components/)

Hope this helps.

### CODE REVIEW

**Comment by WardPearce** [comment]
Please don't pointlessly plug projects not relevant to this repo.
